src/mqt/predictor/rl/tracer.py

```
@@ -0,0 +1,281 @@
+
+"""Visualization module for recording and exporting the RL compilation process."""
+
+from __future__ import annotations
+
+import json
+import time
+from dataclasses import asdict, dataclass, field
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+import qiskit.qasm2 as qasm2
```

Ruff (0.15.9)

[warning] Use `from qiskit import qasm2` in lieu of alias

Replace with `from qiskit import qasm2`

(PLR0402)